src/routes/backoffice.ts

```diff
@@ -67,6 +67,47 @@ export const triggerBookingsWebhook = async (solarisAccountId) => {
   await triggerWebhook(TransactionWebhookEvent.BOOKING, payload);
 };
 
+export const setAccountSeizureProtectionHandler = async (req, res) => {
+  const { email } = req.params;
+
+  const {
+    currentBlockedAmount,
+    protectedAmount,
+    protectedAmountExpiring,
+    protectedAmountExpiringDate,
+  } = req.body;
+
+  const persons = await getAllPersons();
+  const person = persons.find((item) => item.email === email);
+
+  if (!person?.account) return null;
+
+  person.account = {
+    ...person.account,
+    seizure_protection: {
```

Comment:
Seizure protection can be removed. We either need another button on person.html to remove seizure protection for an account, or we can remove the prop if all values are empty. The first option gives more clarity to the user.